runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -325,11 +306,12 @@ void
 MM_StandardAccessBarrier::preBatchObjectStoreImpl(J9VMThread *vmThread, J9Object *dstObject)
 {
 #if defined(OMR_GC_MODRON_CONCURRENT_MARK)
+	Assert_MM_true(!_extensions->usingSATBBarrier());
 	/* Call the concurrent write barrier if required */
 	if(_extensions->concurrentMark && 
 		(vmThread->privateFlags & J9_PRIVATE_FLAGS_CONCURRENT_MARK_ACTIVE) &&
 		_extensions->isOld(dstObject)) {
-		/* Once dependent OMR changes are in, this should be changed to: concurrentPostWriteBarrierStore(vmThread->omrVMThread, dstObject); */
+		/* Once dependent OMR changes are in, this should be changed to: concurrentPreWriteBarrierBatchStore(vmThread->omrVMThread, dstObject); */
```

Comment:
Isn't it vice versa: MM_StandardAccessBarrier::preBatchObjectStoreImpl should be renamed to be _post_?
...in which case J9ConcurrentWriteBarrierBatchStore can be replaced with concurrentPostWriteBarrierStore now.

Reply:
Right, we discussed this before, was confused looking at it

Reply:
Renamed `preBatchObjectStore`/`preBatchObjectStoreImpl` -> `postBatchObjectStore`/`postBatchObjectStoreImpl`